VideoLocker/src/main/java/org/edx/mobile/view/custom/ETitleRowView.java

```diff
@@ -0,0 +1,76 @@
+package org.edx.mobile.view.custom;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.util.AttributeSet;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+import org.edx.mobile.R;
+
+import static org.edx.mobile.R.*;
+
+/**
+ * custom view to support 3 type of text decoration for one row
+ * <li>---Text---</li>
+ * <li>Text------</li>
+ * <li>------Text</li>
+ * The support custom attributes are
+ *   app:title="@string/or_sign_up_with_email"
+ *   app:titleFontSize="8sp"
+ *   app:decorationStyle="left"  -- decorationStyle is enum type, with values {center,left, right}, default is center
+ */
+public class ETitleRowView extends LinearLayout {
```

Comment:
Since this is in the edx package do we need the ``E`` in the name? I guess it refers to ``ETextView`` but maybe we should consider renaming that instead?